notes.md

```diff
@@ -696,12 +700,22 @@ Encryption information is stored in the `ContentEncodings Element` under the `Co
 For encryption systems sharing public/private keys, the creation of the keys and the exchange of keys
 are not covered by this document. They have to be handled by the system using Matroska.
 
+The algorithms described in (#ContentEncAlgoValues) support
+different modes of operations and key sizes.  The specification of these
+parameters is required for a complete solution, but is out of scope of this
+document and left to the proprietary implementations using them or subsequent
+profiles of this document.
+
 The `ContentEncodingScope Element` gives an idea of which part of the track are encrypted.
 But each `ContentEncAlgo Element` and its sub elements like `AESSettingsCipherMode` really
 define how the encrypted should be exactly interpreted.
 
-The AES-CTR system, which corresponds to `ContentEncAlgo` = 5 ((#contentencalgo-element)) and `AESSettingsCipherMode` = 1 ((#aessettingsciphermode-element)),
-is defined in the [@?WebM-Enc] document.
+An example of an extension that builds upon these security-related fields in this specification is [@?WebM-Enc].
+It uses AES-CTR, `ContentEncAlgo` = 5 ((#contentencalgo-element)) and `AESSettingsCipherMode` = 1 ((#aessettingsciphermode-element)).
+
+Matroska Writers **MUST NOT** use insecure cryptographic algorithms to create new
```

Comment:
Isn't "Matroska Writer" the same kind of term as "`Matroska Reader`"? Shouldn't it be `Matroska Writer` (backticked)?

Reply:
It's consistent throughout the whole document, but yes it's one of the terms that should be backticked.